File: ios/ios/NoteTableViewCell.m
//
//  NoteTableViewCell.m
//  ios
//
//  Created by Sahil Ishar on 3/11/16.
//  Copyright © 2016 Kevin Galligan. All rights reserved.
//

#import "NoteTableViewCell.h"

@implementation NoteTableViewCell

- (void)awakeFromNib {
    // Initialization code
    
    [self.imageView setAutoresizingMask:UIViewAutoresizingNone];
    [self.imageView setClipsToBounds:YES];
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: ios/ios/NoteTableViewCell.swift
//
//  NoteTableViewCell.swift
//  ios
//
//  Created by Kachi Nwaobasi on 7/11/17.
//  Copyright © 2017 Kevin Galligan. All rights reserved.
//

import Foundation

class NoteTableViewCell : UITableViewCell {
    
    @IBOutlet weak var noteTitleLabel: UILabel!
    @IBOutlet weak var noteLabel: UILabel!
    @IBOutlet weak var noteImageView: UIImageView!
    
    override func awakeFromNib() {
        // Initialization code
        imageView?.autoresizingMask = UIViewAutoresizing(rawValue: 0)
        imageView?.clipsToBounds = true
    }
}
